ui/pages/confirmation/templates/flask/snap-confirm/index.scss

```diff
@@ -0,0 +1,10 @@
+.snap-confirm {
+  padding: 16px 32px;
+  border-top: 1px solid var(--Grey-100);
+  border-bottom: 1px solid var(--Grey-100);
+  margin: 24px 0 16px 0;
+
+  .text {
+    font-family: monospace;
```

Comment:
Because the scrollable text area is intended for arbitrary text, including e.g. JSON objects to sign. In the future we may make the text style configurable.